CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,6 @@
 ## Purpose
 
-A primary goal of our community, our user groups and mailing lists, symposia and other events is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all.
+A primary goal of our community, our user groups and mailing lists, symposia and other events is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
```

Comment:
Looking at this more closely, I think this addition is unnecessary; the protected characteristics are listed below, and restating a subset here makes it appears as if some characteristics are more important than others. I think it was better as it was originally.

Reply:
I'll remove this, even though I would prefer to keep it. I agree that it is repetitive, but I feel it is important to be explicit in the first paragraph (likely all that anyone will ever read).